mission/para_player_init_client.sqf

```diff
@@ -28,140 +28,191 @@
 		use_paradigm_init = 1;
 */
 
+
+/*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!
+
+after adding any more `call _fnc_do_loading_bar` lines you will
+need to work out how many times `_fnc_do_loading_bar` is called in this file
+
+update the hardcoded `_transitionsTotal` variable if you want to change this
+
+**TIP** -- use this command line
+
+```bash
+# count lines in the file that match the string
+grep -c 'call _fnc_do_loading_bar' mission/para_player_init_client.sqf
+```
+
+!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*!*/
+
+private _transitionsTotal = 25;
+private _loadingChanges = 0;
+private _imgSwitched = false;
+
+private _fnc_do_loading_bar = {
```

Comment:
I don't like this is a dynamically reated function. should be an actual script in the mike force files

Reply:
annoyingly need this for managing the image transitions atm